ui/src/main/java/com/kotcrab/vis/ui/widget/VisImageTextButton.java

```diff
@@ -48,6 +48,8 @@
  * @see Button
  */
 public class VisImageTextButton extends Button implements Focusable, BorderOwner {
+	public enum Orientation { text_right, text_left, text_top, text_bottom }
```

Comment:
In Java, enum constants should typically be written in `UPPER_CASE`. For consistency with e.g. libGDX `Align` enum, you can use `camelCase` instead, but please don't use `snake_case`.

Reply:
Fixed in f92a3ae